src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -184,7 +184,7 @@ public LearningPipeline()
                 {
                     if (lastTransformModel != null)
                         transformModels.Insert(0, lastTransformModel);
-
+                        
                     var modelInput = new Transforms.ModelCombiner
```

Comment:
Somehow lines with whitespace got inserted here semi-randomly.

Reply:
I think it during merge with master. My fault.

---
In reply to: [188484776](https://github.com/dotnet/machinelearning/pull/142#discussion_r188484776) [](ancestors = 188484776)